src/plans/debug.jl

```diff
@@ -399,26 +403,27 @@ how to print the entry.
 # Additional fields
 
 * `field`: symbol the entry can be accessed with within [`AbstractManoptSolverState`](@ref)
+* `at_init`: whether to print also at initialization
 
 # Constructor
 
-    DebugEntry(f; prefix="\$f:", format = "\$prefix %s", io=stdout)
-
+    DebugEntry(f; prefix="\$f:", format = "\$prefix %s", io=stdout, at_init=true)
 """
 mutable struct DebugEntry <: DebugAction
     io::IO
     format::String
     field::Symbol
-    function DebugEntry(f::Symbol; prefix = "$f:", format = "$prefix %s", io::IO = stdout)
-        return new(io, format, f)
+    at_init::Bool
+    function DebugEntry(f::Symbol; prefix = "$f:", format = "$prefix %s", io::IO = stdout, at_init::Bool = true)
+        return new(io, format, f, at_init)
     end
 end
 function (d::DebugEntry)(::AbstractManoptProblem, st::AbstractManoptSolverState, k)
-    (k >= 0) && Printf.format(d.io, Printf.Format(d.format), getfield(st, d.field))
+    (k >= !d.at_init) && Printf.format(d.io, Printf.Format(d.format), getfield(st, d.field))
```

Comment:
My only small semantic remark would be, that “Integer larger or equal a boolean” is not so easy to read/parse.

Could we maybe do something like `(k >= (d.at_init ? 0 : 1))` yes for most interpretations that is the same.

Reply:
Yes, I thought about that. We can certainly use your suggestion. I agree, it's a bit easier to read.